ArtOfIllusion/src/artofillusion/keystroke/KeystrokePreferencesPanel.java

```diff
@@ -133,7 +133,8 @@ private void editRecord()
 
   private void addRecord()
   {
-    KeystrokeRecord record = new KeystrokeRecord(0, 0, "", "", ScriptRunner.LANGUAGES[0]);
+    // Beanshell is the only supported language here 
+    KeystrokeRecord record = new KeystrokeRecord(0, 0, "", "", ScriptRunner.Language.BEANSHELL.name);
```

Comment:
This is not actually true. Both Beanshell and Groovy are fully functional options here. It's just that creating a new record *defaults* to one of them - and, in fact, the main branch defaults to *groovy*. The language can be edited while editing the actual keystroke script.

Reply:
Hey, I'm having trouble understanding what's happening here on GitHub. 
After your comment about the main branch defaulting to Groovy, just to be sure I didn't by mistake change the default to Beanshell, I am trying to check the original version of this file on GitHub, in the master branch of AOI/AOI [here](https://github.com/ArtOfIllusion/ArtOfIllusion/blob/master/ArtOfIllusion/src/artofillusion/keystroke/KeystrokePreferencesPanel.java) and what I'm seeing is changes that I have only introduced in the first commit of my branch (see line 136,     `KeystrokeRecord record = new KeystrokeRecord(0, 0, "", "", ScriptRunner.LANGUAGES[0]);`) for this PR. 
Is this how it's supposed to be? Am I looking at the right place?

Reply:
That line is in the original file. `ScriptRunner` was already using an array for "languages", you had added one for the file extensions.